### ISSUE TITLE
Insufficient Liquidity is not shown on swap button but console logs the error

### ISSUE THREAD

**Issue by srirambv** [brave/brave-browser#20467]
## Description 

Insufficient Liquidity is not shown on swap button but console logs the error

## Steps to Reproduce

   1. Try to swap USDC to DAI

## Actual result:

![image](https://user-images.githubusercontent.com/17010094/149147486-df7be343-66f6-4002-9e8d-7c6750c06abc.png)

## Expected result:
Show `Insufficient Liquidity` message on the swap button instead of spinner icon

## Reproduces how often: 

Easy I suppose

## Brave version (brave://version info)

Brave | 1.36.32 Chromium: 97.0.4692.71 (Official Build) nightly (x86_64 translated)
-- | --
Revision | `adefa7837d02a07a604c1e6eff0b3a09422ab88d-refs/branch-heads/4692@{#1247}`
OS | macOS Version 12.1 (Build 21C52)

## Version/Channel Information:

- Can you reproduce this issue with the current release? Maybe Not sure
- Can you reproduce this issue with the beta channel? Maybe Not sure
- Can you reproduce this issue with the nightly channel? Yes

## Other Additional Information:

- Does the issue resolve itself when disabling Brave Shields? NA
- Does the issue resolve itself when disabling Brave Rewards? NA
- Is the issue reproducible on the latest version of Chrome? NA

## Miscellaneous Information:

cc: @onyb

**Comment by srirambv** [comment]
Yes this seems to be fixed as well. Not seeing any errors on console or insufficient liquidity message for swap 
![image](https://user-images.githubusercontent.com/17010094/150345405-d3aa1928-c039-469d-b6a6-f9419896ded4.png)

**Comment by onyb** [comment]
Seeing this on nightly again, so going to reopen this issue. The issue probably disappeared because USDC-DAI liquidity was back on Ropsten (I recall the issue was open for a while).

To reproduce now, perform an ETH-USDC swap, with From amount to be `1`.

https://user-images.githubusercontent.com/3684187/152545334-0d21e09c-8254-49d5-859f-9225828cbbf1.mov

**Comment by Uni-verse** [comment]
Verification In progress using

```
Brave	1.37.105 Chromium: 100.0.4896.46 (Official Build) (x86_64) 
Revision	5ca33821b2211805855c77d334353d27c616a7ca-refs/branch-heads/4896@{#584}
OS	macOS Version 12.1 (Build 21C52)
```
